### DISCUSSION TITLE
LibLZ4 CMake Question: where is LZ4Config.cmake?

### DISCUSSION THREAD

**Discussion by JeffreyO** [termux/termux-packages#10798] [Q&A] [answered]
It looks like the 1.9.3 tag doesn't have the right file. Though. would it be possible just to use the [LZ4Config.cmake.in](https://github.com/lz4/lz4/blob/dev/build/cmake/lz4Config.cmake.in) file with the current source code?

**Comment by sylirre** [comment, marked as answer]
Added in https://github.com/termux/termux-packages/commit/c32bc2b40b5ed0ef1e993e19b5585bab1d4384d3
